src/System.Management.Automation/FormatAndOutput/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -646,6 +650,25 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Proc
                 .EndTable());
         }
 
+        private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Process_IncludeCommandLine()
+        {
+            yield return new FormatViewDefinition("ProcessWithUserName",
```

Comment:
Should be called `ProcessWithCommandLine`

Reply:
Fixed.